### PR TITLE
windows setup

### PR DESCRIPTION
**Pull request by r0fls**
Implements idea from https://github.com/channelcat/sanic/pull/507. Also includes a 🚀

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,33 +15,47 @@
...
         raise RuntimeError('Unable to determine version.')
 
-install_requires = [
-    'httptools>=0.0.9',
-    'ujson>=1.35',
-    'aiofiles>=0.3.0',
-    'websockets>=3.2',
-]
+setup_kwargs =  {
+        'name': 'sanic',
```
Comment (messense) [inline setup.py]: The indent should be four spaces.

Comment (messense) [inline setup.py]: Please consider setup [Hound CI](https://houndci.com/) for automated code style review.

Comment (seemethere) [inline setup.py]: The setup.py isn't covered by the flake8 check that we run.

Comment (messense) [inline setup.py]: @seemethere It would be much nicer to let Hound CI submit reviews to pull requests than looking into Travis CI logs.

Comment (seemethere) [inline setup.py]: I can understand the reasoning but I don't want to add another dependency in our development pipeline. 

I'll look into Hound CI but chances are it'll stay how it is. Also we could look into something like PEP8 Speaks or something else.

Comment (messense) [inline setup.py]: Integrate with PEP8 Speaks would be great too.

**Pull request merged**